ldsc/src/main/resources/regionsToAnnot.py

```diff
@@ -15,57 +15,92 @@
-    file = f'{s3_in}/out/ldsc/regions/{sub_region}/merged/{region_name}/{region_name}.csv'
-    subprocess.check_call(['aws', 's3', 'cp', file, f'./{region_name}/'])
+    file = f'{s3_in}/out/ldsc/regions/merged/{sub_region}/{region_name}/{region_name}.csv'
+    subprocess.check_call(['aws', 's3', 'cp', file, f'./{sub_region}/{region_name}/'])
 
 
-def convert_to_bed(region_name):
-    with open(f'./{region_name}/{region_name}.bed', 'w') as f_out:
-        with open(f'./{region_name}/{region_name}.csv', 'r') as f_in:
-            line = f_in.readline()
-            while len(line) > 0:
-                filtered_line = '\t'.join(line.split('\t')[:3])
-                f_out.write(f'chr{filtered_line}\n')
-                line = f_in.readline()
+def parse_range_line(line):
+    chromosome, pos_start_str, pos_end_str = line.split('\t')[:3]
+    return chromosome, int(pos_start_str), int(pos_end_str)
 
 
-def make_annot(region_name, ancestry, CHR):
+def get_parsed_range_line(range_f):
+    line = range_f.readline().strip()
+    if len(line) == 0:
+        return None, None, None
+    else:
+        return parse_range_line(line)
+
+
+def set_range_start(range_f, g1000_chr):
+    range_chr, pos_start, pos_end = get_parsed_range_line(range_f)
+    while range_chr is not None and range_chr != g1000_chr:
+        range_chr, pos_start, pos_end = get_parsed_range_line(range_f)
+    return range_chr, pos_start, pos_end
+
+
+def parse_g1000_line(line):
+    chromosome, _, _, position_str = line.split('\t')[:4]
+    return chromosome, int(position_str)
+
+
+def get_parsed_g1000_line(g1000_f):
+    line = g1000_f.readline().strip()
+    if len(line) == 0:
+        return None, None
+    else:
+        return parse_g1000_line(line)
+
+
+def make_annot(sub_region, region_name, ancestry, CHR):
     print(f'Making annot for {region_name}, ancestry: {ancestry}, chromosome: {CHR}')
-    subprocess.check_call([
-        'python3', f'{ldsc_files}/make_annot.py',
-        '--bed-file', f'{region_name}/{region_name}.bed',
-        '--bimfile', f'{g1000_files}/{ancestry}/chr{CHR}.bim',
-        '--annot-file', f'./{region_name}/{ancestry}/annot/{region_name}.{CHR}.annot.gz'
-    ])
+    with gzip.open(f'{sub_region}/{region_name}/{ancestry}/annot/{region_name}.{CHR}.annot.gz', 'w') as annot_f:
+        annot_f.write(b'ANNOT\n')
+        with open(f'{g1000_files}/{ancestry}/chr{CHR}.bim', 'r') as g1000_f:
+            with open(f'./{sub_region}/{region_name}/{region_name}.csv', 'r') as range_f:
+                g1000_chr, position = get_parsed_g1000_line(g1000_f)
+                range_chr, pos_start, pos_end = set_range_start(range_f, g1000_chr)
+                while range_chr is not None and g1000_chr is not None and g1000_chr == range_chr:
+                    if position <= pos_start:
+                        annot_f.write(b'0\n')
+                        g1000_chr, position = get_parsed_g1000_line(g1000_f)
+                    elif position <= pos_end:
```

Comment:
while we are after the start, but less than the end (inclusive): write a 1 and get a new g1000 line